packages/core/i18n/core-i18n-browser/src/index.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export type { I18nStart } from './types';
```

Comment:
aside (not specific to this PR): It seems like overkill to have a whole package just for a single type. However, that's the way we chose to do things.

Reply:
Yes it might seem redundant but it keeps things consistent and also separates the external contract from the internal implementation.